src/style.css

```
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+
+}
+
+.cont__block {
+  flex-grow: 1;
+  flex-basis: 300px;
+  height: 300px;
+  line-height: 300px;
+  text-align: center;
+}
+
+.container > div:nth-child(1) {
```

use classnames for styling
```suggestion
.container > cont__block:nth-child(1) {
```